.aios-core/infrastructure/scripts/llm-routing/templates/deepseek-proxy.sh

```
@@ -0,0 +1,65 @@
+        sleep 1
+        if curl -s "http://127.0.0.1:$PROXY_PORT/health" > /dev/null 2>&1; then
+            echo "Proxy started on port $PROXY_PORT (PID: $(cat $PID_FILE))"
+        else
+            echo "Failed to start proxy. Check /tmp/deepseek-proxy.log"
+        fi
+        ;;
+
+    stop)
+        echo "Stopping proxy..."
+        if [ -f "$PID_FILE" ]; then
+            kill $(cat "$PID_FILE") 2>/dev/null
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)